fmriprep/workflows/bold/confounds.py

```diff
@@ -71,20 +79,25 @@ def init_bold_confs_wf(bold_file_size_gb, use_aroma, ignore_aroma_err, metadata,
 
     **Inputs**
 
-        bold_t1
-            BOLD image, resampled in T1w space
+        bold
+            BOLD image, after the prescribed corrections (STC, HMC and SDC)
+            when available.
+        bold_mask
+            BOLD series mask
         movpar_file
             SPM-formatted motion parameters file
         t1_mask
             Mask of the skull-stripped template image
         t1_tpms
             List of tissue probability maps in T1w space
-        bold_mask_t1
-            BOLD series mask in T1w space
+        t1_bold_xform
+            Affine matrix that maps the T1w space into alignment with
+            the native BOLD space
```

Comment:
I think you're going to re-introduce issues with partial-FoV BOLD series if you're not masking with the BOLD mask after resampling the masks created in T1w space. (I think it's fine to use this transform. But I don't think it replaces masking.)

Reply:
Oh, I see. I'll make sure masks do not go off limits with partial-FoV in BOLD space.

Reply:
Done

Reply:
Actually, I'm thinking this doesn't make any difference: we work on BOLD space (so it is still partial if it was partial in first place). Nothing will be considered if it is off FoV.

Reply:
Yeah, that seems correct. I suppose I was thinking we were in `bold_space-T1w` still.